### PR TITLE
Feature/readme documentation

### PR DESCRIPTION
**Pull request by magnificode**
# Describe your PR

Adds additional contribution guidelines and high level process description for the project.

Related to # N/a 
Fixes # N/a

## Pages/Interfaces that will change

N/a

### Screenshots / video of changes

N/a

## Steps to test

N/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,95 @@
-# RebuildBlackBusiness.com
+<h1 align="center">Welcome to Rebuild Black Business 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
+  <a href="https://twitter.com/ThugDebugger" target="_blank">
```
Comment (jlengstorf) [inline README.md:4]: without context, it’s unclear why this website is promoting one person’s Twitter — should that be clarified? is there a RebuildBlackBusiness Twitter account that might be more appropriate?

File: README.md [hunk 2]
```diff
@@ -1,32 +1,95 @@
...
 
-Second, clone your copy of the repo to your local machine:
+## 🤝 Contributing
 
-```bash
-# clone the repo
-git clone git@github.com:YOUR_USERNAME_HERE/RBB-Website.git
+Contributions, issues and feature requests are welcome!<br />Feel free to check [issues page](https://github.com/Rebuild-Black-Business/RBB-Website/issues).
+
+### How can I get involved now?
```
Comment (jlengstorf) [inline README.md:17]: There's no mention of the Discord here. If the process is designed to require devs to go through Discord to get access, we should probably call out that process first.

File: README.md [hunk 3]
```diff
@@ -1,32 +1,95 @@
...
+
+We're tracking all high level Web Development tasks in Trello:
+
+- [Front-End Development Trello](https://trello.com/b/d1sguaj6/rebuild-black-business-frontend-dev)
+- [Back-End Development Trello](https://trello.com/b/034hEV4X/rebuild-black-business-backend-dev)
```
Comment (jlengstorf) [inline README.md:22]: Feel free to ignore this, but don't GitHub Issues/Projects do everything Trello’s doing here in a way that doesn't require signing up for additional accounts? Another benefit of using issues is that they can be referenced from pull requests to add context, auto-move between swim lanes, etc.

File: README.md [hunk 4]
```diff
@@ -1,32 +1,95 @@
...
+
+To get access to this data, please join the [Rebuild Black Business Discord](https://discord.gg/272XMuv) and express your interest in helping to your assigned Team Lead. They'll get you read access to the data in a timely manner.
+
+## Install
+
+**Clone the repository**
+
+```sh
+# clone the repo
+git clone git@github.com:Rebuild-Black-Business/RBB-Website.git
```
Comment (jlengstorf) [inline README.md:79]: Requiring people to already have an invite before being able to contribute seems like a blocker for community contributions. Making it as simple as possible for someone to quickly fork/fix/PR a small contribution seems like a win — not everyone will be able/willing to actually join in, but many will do hotfixes as they see them if it doesn't require jumping through lots of hoops.

File: README.md (hunk 1 above)
Comment (magnificode) [reply README.md:4]: Great point, I'll remove this for now until we have unified branding for Rebuild Black Business!

File: README.md (hunk 3 above)
Comment (magnificode) [reply README.md:22]: 💯 - I think we'll see what we can do about using GitHub as a more single source of truth for the roadmap and current workload. Trello is an artifact of just trying to get up and running quickly. We'll simplify this as quickly as we can!

File: README.md (hunk 2 above)
Comment (magnificode) [reply README.md:17]: 👍 - I'll add a paragraph here explaining the role Discord plays!

File: README.md (hunk 4 above)
Comment (magnificode) [reply README.md:85]: Fair point for sure, we'll maintain the forking structure for pull request to reduce the barrier to entry.

File: README.md (hunk 3 above)
Comment (racedale) [reply README.md:21]: Now that we have a repo I'd be happy to start creating new pieces of work in GitHub

**Pull request merged**